# Awesome LatchBio [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)

> Bioinformatics is an interdisciplinary field that develops methods and software tools for understanding biological data. â€” [Wikipedia](https://en.wikipedia.org/wiki/Bioinformatics)

A curated list of awesome Bioinformatics pipelines and software available on [LatchBio](https://console.latch.bio/explore). Command line or programmatic tools are now with Graphical User Interface and can be run without the hassle of software installation! Please feel free to [contribute](CONTRIBUTING.md)!

<!-- START doctoc generated TOC please keep comment here to allow auto update -->
<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
## Table of Contents

- [Genome Assembly](#genome-assembly)
  - [De novo](#de-novo)
  - [Metrics](#metrics)
- [Multiple sequence alignment](#multiple-sequence-alignment)
  - [Align](#align)
  - [Trimming](#trimming)
  - [Summary statistics](#summary-statistics)
- [Phylogenetics](#phylogenetics)
  - [Phylogenetic inference](#phylogenetic-inference)
- [Codon analyses](#codon-analyses)
  - [Optimization](#optimization)
- [CRISPR](#crispr)
  - [CRISPR locus](#crispr-locus)
- [Contributors](#contributors)

<!-- END doctoc generated TOC please keep comment here to allow auto update -->

---

### Genome Assembly

#### De novo

- **[shasta](https://github.com/chanzuckerberg/shasta)** 
  <br/>
  **Shasta** | [latch source code](https://github.com/nahid18/shasta) | [paper-2020](https://pubmed.ncbi.nlm.nih.gov/32686750) | [documentation](https://chanzuckerberg.github.io/shasta)
  - Rapidly produce accurate assembled sequence using DNA reads generated by Oxford Nanopore flow cells as input.

#### Metrics
- **[BioKIT genome assembly metrics](https://jlsteenwyk.com/BioKIT/)**
  <br/>
  **wf.genome_assembly_metrics_biokit** | [paper-2022](https://academic.oup.com/genetics/advance-article-abstract/doi/10.1093/genetics/iyac079/6583183) | [documentation](https://jlsteenwyk.com/BioKIT/)
  - Calculate diverse summary statistics for a genome assembly using [BioKIT](https://academic.oup.com/genetics/advance-article-abstract/doi/10.1093/genetics/iyac079/6583183) 

### Multiple sequence alignment

#### Align
- **[MAFFT](https://mafft.cbrc.jp/alignment/software/)**
  <br/>
  **wf.align_sequences_mafft** | [paper-2013](https://academic.oup.com/mbe/article/30/4/772/1073398) | [documentation](https://mafft.cbrc.jp/alignment/software/))
  - Multiple sequence alignment program.

- **[CIAlign](https://github.com/KatyBrown/CIAlign.git)**
  <br/>
  **wf.CIAlign** | [Paper](https://peerj.com/articles/12983/) | [Documentation](https://github.com/GeOdette/clalign.git)
  - Clean, visualise, and interpret multiple sequence alignment.

#### Trimming
- **[ClipKIT](https://github.com/JLSteenwyk/ClipKIT)** 
  <br/>
  **wf.clipkit** | [paper-2020](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3001007) | [documentation](https://jlsteenwyk.com/ClipKIT/)
  - Multiple sequence alignment trimming for accurate phylogenomic inference.


- **[CIAlign](https://github.com/KatyBrown/CIAlign.git)
  <br/>
  **wf.CIAlign** | [Latch source code](https://github.com/GeOdette/clalign.git) | [Documentation](https://github.com/GeOdette/clalign.git)
  - Clean, visualise, and interpret the multiple sequence alignment

#### Summary statistics
- **[BioKIT alignment_summary_statistics](https://jlsteenwyk.com/BioKIT/)**
  <br/>
  **wf.alignment_summary_statistics** | [paper-2022](https://academic.oup.com/genetics/advance-article-abstract/doi/10.1093/genetics/iyac079/6583183) | [documentation](https://jlsteenwyk.com/BioKIT/)
  - Calculate diverse summary statistics for a multiple sequence alignment using [BioKIT](https://academic.oup.com/genetics/advance-article-abstract/doi/10.1093/genetics/iyac079/6583183)

### Phylogenetics

#### Phylogenetic inference
- **[IQ-TREE2](http://www.iqtree.org/doc/)** 
  <br/>
  **wf.infer_phylogeny** | [paper-2020](https://doi.org/10.1093/molbev/msaa015) | [documentation](http://www.iqtree.org/doc/)
  - Efficient software for phylogenomic inference.

### Codon analyses

#### Optimization
- **[BioKIT gRSCU](https://jlsteenwyk.com/BioKIT/)** 
  <br/>
  **wf.codon_optimization_biokit** | [paper-2022](https://academic.oup.com/genetics/advance-article-abstract/doi/10.1093/genetics/iyac079/6583183) | [documentation](https://jlsteenwyk.com/BioKIT/)
  - Estimate codon optimization using [BioKIT](https://academic.oup.com/genetics/advance-article-abstract/doi/10.1093/genetics/iyac079/6583183)

### CRISPR
#### CRISPR locus
- **[CRISPRbuilderTB](https://github.com/GeOdette/crisptb.git)**
  <br/>
  **wf.CRISPRbuilderTB** | [Paper](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008500) | [documentation](https://github.com/GeOdette/crisptb.git)
  - Reconstruct Mycobacterium tuberculosis CRISPR-Cas loci from short reads using [CRISPRbuilder-TB](https://github.com/cguyeux/CRISPRbuilder-TB.git)

### Contributors
![Your Repository's Stats](https://contrib.rocks/image?repo=nahid18/Awesome-LatchBio)
